code/SoloSavingsApp/src/main/resources/application.properties

```diff
@@ -10,4 +10,7 @@ spring.jpa.show-sql: true
 
 server.port=8888
 server.error.path=/error
-logging.level.org.springframework.security=TRACE
\ No newline at end of file
+logging.level.org.springframework.security=TRACE
+
+jwt.secret=mysecretkey
```

Comment:
Going forward we probably need another way to manage this secret. But that is likely after initial JWT implementation.